recipes/libacl/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+class LibAclConan(ConanFile):
+    name = "libacl"
```

Comment:
```suggestion
    name = "libacl"
```

acl seems to be far more common name:
https://repology.org/project/acl/versions
https://repology.org/project/libacl/versions

Reply:
i would change the name to acl - shall i stop the PR then and start a new one so the folder can be changed from libacl to acl?

Reply:
i would first change the name in conanfile.py and you guys then can inform me if you would prefer renaming to folder too. i think for "clean style" i should change the foldername too

Reply:
No need to open a new PR

Reply:
wow holy! :D i changed the foldername and jenkins still accept. 👍  
I was little worry that the servers maybe thinks that i push now a 2nd package in this PR. but nope - all ok. 
Cool - thanks!